OneBusAway/ui/trip_details/OBAArrivalAndDepartureViewController.m

```diff
@@ -175,12 +172,54 @@ - (void)viewWillDisappear:(BOOL)animated {
     [[OBAHandoff shared] stopBroadcasting];
 }
 
+- (void)viewWillLayoutSubviews {
+    [super viewWillLayoutSubviews];
+    [self.mapController.view mas_makeConstraints:^(MASConstraintMaker* make) {
+        make.edges.equalTo(self.collapsingHeaderView);
+    }];
+    
+    [self.collapsingHeaderView mas_makeConstraints:^(MASConstraintMaker* mas) {
+        mas.left.equalTo(self.view.mas_left);
```

Comment:
Masonry has a neat chaining syntax that'll let you collapse the first part down into:

`mas.left.right.and.top.equalTo(self.view)`